### ISSUE TITLE
sage.{rings,modules,geometry}: Replace imports from sage.*.all for namespace packages

### ISSUE THREAD

**Issue by mkoeppe** [sagemath/sage#34947]
<div id="comment:0"></div>

Using `./sage -fiximports` from #34945.

Also remove trailing whitespace in the affected files.

Part of Meta-ticket #34201

CC:  @alexchandler100 @tscrim @videlec

Component: **refactoring**

Author: **Alex Chandler**

Branch/Commit: **[u/mkoeppe/sage__rings_modules_geometry___replace_imports_from_sage___all_for_namespace_packages](https://github.com/sagemath/sagetrac-mirror/tree/u/mkoeppe/sage__rings_modules_geometry___replace_imports_from_sage___all_for_namespace_packages) @ [`9a212f4`](https://github.com/sagemath/sagetrac-mirror/commit/9a212f42599ce67ca172a431a585a5d680dc642f)**

Reviewer: **Travis Scrimshaw, Matthias Koeppe**

_Issue created by migration from https://trac.sagemath.org/ticket/34947_

**Comment by mkoeppe** [comment]
Branch: **[u/mkoeppe/sage__rings_modules_geometry___replace_imports_from_sage___all_for_namespace_packages](https://github.com/sagemath/sagetrac-mirror/tree/u/mkoeppe/sage__rings_modules_geometry___replace_imports_from_sage___all_for_namespace_packages)**

**Comment by mkoeppe** [comment]
<div id="comment:2"></div>

New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/f1b7dc1c896fceff044e4a267848f1a1cd18752f"><code>f1b7dc1</code></a></td><td><code>sage -fiximports src/sage/{rings,modules,geometry}</code></td></tr></table>

**Comment by mkoeppe** [comment]
Commit: **[`f1b7dc1`](https://github.com/sagemath/sagetrac-mirror/commit/f1b7dc1c896fceff044e4a267848f1a1cd18752f)**

**Comment by mkoeppe** [comment]
Reviewer: **tscrim, vdelecroix**

**Comment by mkoeppe** [comment]
Changed reviewer from **tscrim, vdelecroix** to none

**Comment by mkoeppe** [comment]
<div id="comment:5" align="right">comment:5</div>

Lint and Build&Test failures are unrelated.

**Comment by tscrim** [comment]
<div id="comment:7" align="right">comment:7</div>

One essentially trivial thing:

```diff
-from sage.arith.misc import factor
-from sage.arith.misc import primitive_root
-from sage.arith.misc import CRT_basis
+from sage.arith.misc import factor, primitive_root, CRT_basis
```
(I am not a fan of the `from sage.rings.integer_ring import Z as ZZ` whereas `ZZ` is included there, but that is even more trivial than the above.)
Modulo that LGTM.

**Comment by tscrim** [comment]
Reviewer: **Travis Scrimshaw**

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
Changed commit from **[`f1b7dc1`](https://github.com/sagemath/sagetrac-mirror/commit/f1b7dc1c896fceff044e4a267848f1a1cd18752f)** to **[`9a212f4`](https://github.com/sagemath/sagetrac-mirror/commit/9a212f42599ce67ca172a431a585a5d680dc642f)**

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
<div id="comment:8"></div>

Branch pushed to git repo; I updated commit sha1. New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/6cceac972d37b7d12635d36fc31b91d1efa3b9da"><code>6cceac9</code></a></td><td><code>src/sage/rings/polynomial: Consolidate some imports from the same module</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/9a212f42599ce67ca172a431a585a5d680dc642f"><code>9a212f4</code></a></td><td><code>git grep -l -E ' (Q as QQ|Z as ZZ)' | xargs sed -i.bak 's/ Q as QQ/ QQ/;s/ Z as ZZ/ ZZ/;'</code></td></tr></table>

**Comment by mkoeppe** [comment]
Changed reviewer from **Travis Scrimshaw** to **Travis Scrimshaw, Matthias Koeppe**

**Comment by tscrim** [comment]
<div id="comment:10" align="right">comment:10</div>

Thanks.

**Comment by mkoeppe** [comment]
<div id="comment:11" align="right">comment:11</div>

Thanks, Travis!

**Comment by mkoeppe** [comment]
Merged in 10.0.beta0
